mod-sender/src/main/resources/domain/mod-sender/features/sender.feature

```diff
@@ -9,4 +9,16 @@ Feature: Sender - message delivery
     Given path 'message-delivery'
     And request "{}"
     When method POST
-    Then status 422
\ No newline at end of file
+    Then status 422
+
```

Comment:
Please add a test for case when requested user is not found (mod-users returns 404). See `SenderServiceImpl#lookupUser`.